src/cgutils.cpp

```diff
@@ -1075,6 +1075,9 @@ static Value *emit_typeptr_addr(Value *p)
     return emit_nthptr_addr(p, -offset);
 }
 
+static Value *boxed(const jl_cgval_t &v, jl_codectx_t *ctx, bool gcooted=true);
+static Value *boxed(const jl_cgval_t &v, jl_codectx_t *ctx, jl_value_t* type) = delete; // C++11 (temporary to prevent rebase error)
```

Comment:
We only need a pre-C++11 compiler when building against LLVM 3.3, so if this is behind enough ifdefs to not break that then I think we're fine. Hopefully compiler performance will be at parity soon enough that we could confidently remove the LLVM 3.3 code.